src/Simplex/Messaging/Agent.hs

```diff
@@ -402,16 +409,24 @@ deleteConnectionAsync' c@AgentClient {subQ} corrId connId =
     notifyDeleted :: m ()
     notifyDeleted = atomically $ writeTBQueue subQ (corrId, connId, OK)
 
+-- | Add connection to the new receive queue
+switchConnection' :: AgentMonad m => AgentClient -> ConnId -> m ()
```

Comment:
maybe this should have ACorrId and communicate "rotation initiated" to client? may be then communicated to ui for easier debugging

Reply:
and maybe further rotation steps could notify too

Reply:
all not necessary though